Changelog.md

```diff
@@ -13,6 +13,8 @@
 
 -   In some cases, the **Refresh dashboard** button was visible after
     deselecting device.
+-   Was not able to copy modem firmware zip if target directory did not exist,
+    meaning that programming modem firmware failed.
```

Comment:
Think this is a bit too verbose for customers (implementation details)

```suggestion
-   Programming modem firmware failed if target directory did not exist.
```

Maybe this too contains details that are irrelevant to the customer (target directory).
Did this happen randomly? Or on new installs? If so I would say it's sufficient to explain that this behavior was present and is now fixed.